deployment/clip_demographics.sql

```diff
@@ -0,0 +1,55 @@
+-- Function to clip demographics features to region boundaries.
+-- Proportionally allocates values to remaining area of clipped feature.
+
+DROP FUNCTION IF EXISTS ClipDemographics();
+
+CREATE OR REPLACE FUNCTION ClipDemographics()
+  RETURNS void AS
+$$
+DECLARE
+  region geometry;
+  diff geometry;
+  intersection geometry;
+  min_feature_id int4;
+  max_feature_id int4;
+  feature_area NUMERIC;  -- in SRID units
+  intersection_area NUMERIC;
+  contained_proportion NUMERIC;
+BEGIN
+  RAISE INFO 'Going to clip demographics features to region boundary.';
+  region := geom FROM datasources_boundary WHERE id = (SELECT region_boundary_id FROM transit_indicators_otiindicatorsconfig);
```

Comment:
It might be good to fall back to the urban boundary if the user hasn't uploaded a regional boundary at the time this is run.